src/nvim/eval.c

```diff
@@ -10765,32 +10765,40 @@ static void f_getcompletion(typval_T *argvars, typval_T *rettv, FunPtr fptr)
     options |= WILD_KEEP_ALL;
   }
 
+  if (argvars[0].v_type != VAR_STRING || argvars[1].v_type != VAR_STRING) {
+    EMSG(_(e_invarg));
+    return;
+  }
+
   ExpandInit(&xpc);
   xpc.xp_pattern = get_tv_string(&argvars[0]);
```

Comment:
this is already done by `set_one_cmd_context`, so this is a redundant call to `get_tv_string`.